### PR TITLE
ISSUE 125 : disabling network command while running with podman

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: skipper/runner.py [hunk 1]
```diff
@@ -149,7 +151,6 @@ def _network_exists(net):
     cmd = ['network', 'ls']
     if utils.get_runtime_command() == "docker":
         cmd.extend(["-f", "NAME=%s" % net])
-    else:
```
Comment (eranco74) [inline skipper/runner.py:152]: No need for the if as well

File: skipper/data/skipper-entrypoint.sh [hunk 2]
```diff
@@ -1,29 +1,32 @@
 #!/usr/bin/env bash
 
-SKIP_HOME_DIR_PARAM=""
-# if home directory already exists, useradd should not try to create it
-[ -d /home/${SKIPPER_USERNAME} ] && SKIP_HOME_DIR_PARAM="--no-create-home"
+if ! [ -z "${SKIPPER_DOCKER_GID}" ];then
```
Comment (tsorya) [inline skipper/data/skipper-entrypoint.sh:3]: Podman is setting user byitself, so no need for any operation while we are using podman.
SKIPPER_DOCKER_GID is set only if we run with docker

File: skipper/data/skipper-entrypoint.sh [hunk 3]
```diff
@@ -1,29 +1,32 @@
...
   fi
-fi
 
-if grep -q docker /etc/group; then
-   usermod -G root,docker ${SKIPPER_USERNAME}
-else
-   usermod -G root ${SKIPPER_USERNAME}
-fi
+
+  if grep -q docker /etc/group; then
```
Comment (romfreiman) [inline skipper/data/skipper-entrypoint.sh:22]: mm but if it's running by podman, why to touch?

Comment (tsorya) [reply skipper/data/skipper-entrypoint.sh:22]: it will not run with podman

**Comment by tsorya** [comment]
moved to upstream

**Pull request closed without merging**